# HG changeset patch
# User andrew
# Date 1518669922 0
#      Thu Feb 15 04:45:22 2018 +0000
# Node ID adaf109718c10888cce5b6e73af7f3e15a7ab0db
# Parent  3ade0115344b77e6d00462044e0cf68722685bfe
PR3519: Fix further functions with a missing return value.

diff --git openjdk.orig/hotspot/src/cpu/aarch64/vm/c1_LIRAssembler_aarch64.cpp openjdk/hotspot/src/cpu/aarch64/vm/c1_LIRAssembler_aarch64.cpp
--- openjdk.orig/hotspot/src/cpu/aarch64/vm/c1_LIRAssembler_aarch64.cpp
+++ openjdk/hotspot/src/cpu/aarch64/vm/c1_LIRAssembler_aarch64.cpp
@@ -205,6 +205,7 @@
       return Address(base, tmp, Address::lsl(addr->scale()));
     }
   }
+  return Address();
 }
 
 Address LIR_Assembler::as_Address_hi(LIR_Address* addr) {
